testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/AbstractTestHiveViews.java

```diff
@@ -443,6 +443,8 @@ public void testCurrentUser()
     }
 
     @Test(groups = HIVE_VIEWS)
+    // Test is currently flaky on CDH5 environment
+    @Flaky(issue = "https://github.com/trinodb/trino/issues/9074", match = "Error while processing statement: FAILED: Execution Error, return code 2 from org.apache.hadoop.hive.ql.exec.mr.MapRedTask")
```

Comment:
Can we increase the number of retries ? Last time when we run a stress check it failed during the second run too.

Reply:
i wish we could.

Reply:
But we could increase it by updating this value - https://github.com/trinodb/trino/blob/bb06a1a2fef5ba5ca919e01369a05bb6979a2c02/testing/trino-testing-services/src/main/java/io/trino/testng/services/FlakyTestRetryAnalyzer.java#L45